VTUL/iawa
# LIBTD-1656: Finalize controlled vocabularies

**Finalize controlled vocabularies for releasing into production.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1656) (:star:)

* The finalized list for controlled vocabulary is in the IAWA service page: https://webapps.es.vt.edu/confluence/display/LIBTD/IAWA

# What does this Pull Request do? (:star:)
Update controlled vocabularies to the finalized list for production use.

# What's the changes? (:star:)

* Add a unique index in the database table to ensure the uniqueness validation in the model;
* Kept all the terms consistent in a way that subsequent words begins with lower-case character such as ""Color pencil" vs "Colored pencil", "Commercial and office" vs "Commercial and Office" and "Residential-Single-family" vs "Residential-single-family";
* Added new terms such as "Roofs", "Reflected ceiling plans" and "Framing plans".

# How should this be tested?

* Log in as admin user;
* Go to Dashboard --> CONFIGURATION --> Vocabs --> List Vocabs;
* Compare the vocabularies with the finalized list at https://webapps.es.vt.edu/confluence/display/LIBTD/IAWA;
* Note that there are two terms "Interior design" for type and tags, respectively;
* Especially look for multi-word vocabulary terms to see if the subsequent words starts with lower-case letter;
* 45 resource types, 19 mediums, and 36 tags.

# Additional Notes:

* Test this PR in branch LIBTD-1656.

# Interested parties
Tag (@shabububu ) interested parties

(:star:) Required fields
